rasa_core/training/interactive.py

```diff
@@ -588,6 +594,8 @@ def _ask_if_quit(sender_id: Text, endpoint: EndpointConfig) -> bool:
     elif answer == "undo":
         raise UndoLastStep()
     elif answer == "fork":
+        # a fork is created and the first story line saved as if quit
+        _write_story_write_nlu(sender_id, endpoint)
```

Comment:
I do not think this is a good idea from the usability perspective: this will require the user to enter the export info every time they fork a conversation. rather I'd suggest to use the `sender_ids` that we are collecting anyways in `record_messages` to dump all the conversation at once on quit.

Reply:
good point, didn't think of that one

Reply:
I'll change that